lib/data_update_scripts/20200901040521_create_profile_fields.rb

```diff
@@ -1,10 +1,7 @@
 module DataUpdateScripts
   class CreateProfileFields
     def run
-      # NOTE: the CSV importer uses find_or_create_by for both fields and
-      # groups, so this operation is idempotent.
-      csv = Rails.root.join("lib/data/dev_profile_fields.csv")
-      ProfileFields::ImportFromCsv.call(csv)
+      # This is no longer needed
```

Comment:
I'm not 100% sure what the best way forward here is. We had several discussion about no longer removing DUS but once the CSV file is deleted there's nothing here this script could do so I turned it in a no op.